components/login/provider/index.test.jsx

```diff
@@ -94,7 +94,9 @@ describe("ProviderLogin form", () => {
       label,
     });
 
-    const checkOidcSupportSpy = jest.spyOn(useClientId, "checkOidcSupport");
+    const checkOidcSupportSpy = jest
+      .spyOn(useClientId, "checkOidcSupport")
+      .mockImplementation(async () => true);
```

Comment:
Doesn't really matter, but good to know that you can do this too :)

```suggestion
      .mockResolvedValue(true);
```